athenz/assertion.go

```diff
@@ -90,11 +171,88 @@ func resourceAssertionSchema() *schema.Schema {
 					Optional: true,
 					Default:  false,
 				},
+				"id": {
+					Type:     schema.TypeInt,
+					Computed: true,
+				},
+				"condition": {
+					Type:     schema.TypeSet,
+					MaxItems: 2, // each assertion represent acl policy. Since in a given service,
+					// you can make the acl policy enforced on some hosts and not on others,
+					// therefor, to apply more than 2 conditions per assertion doesn't make sense
```

Comment:
`therefore`
what happens if there are more than 2 items?

Reply:
You will get an error during the plan stage